source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -577,6 +580,76 @@ AdaptiveNamespaceStart
         interface IAdaptiveCardElement;
     };
 
+    [
+        version(NTDDI_WIN10_RS1),
+#ifdef ADAPTIVE_CARDS_WINDOWS
+        uuid(05F2F77A-D458-4F73-AAF7-06B91D4C37E6),
+        contract(InternalContract, 1),
+        internal
```

Comment:
Should this be exclusiveto?

Reply:
Yes, fixed.

---
In reply to: [190355696](https://github.com/Microsoft/AdaptiveCards/pull/1490#discussion_r190355696) [](ancestors = 190355696)